src/Components/EmailList/index.js

```diff
@@ -0,0 +1,32 @@
+import EmailPreview from "../EmailPreview"
+import NoEmails from "../NoEmails"
+import "./styles.scss"
+
+const EmailList = ({ emails }) => {
+    return (
+        <div className="listContainer vh-100 overflow-auto">
+            {
+                !emails || emails.length == 0 &&
+                <NoEmails />
+            }
+
+            {
+                emails &&
+                (<>{emails.map((email) => (
+                    <EmailPreview
+                        key={email.id}
+                        id={email.id}
+                        name={email.name}
+                        emailAddress={email.email}
+                        subject={email.subject}
+                        dateCreated={email.date_created}
+                        read={email.read}
+                        bodyPreview={email.body}
+                    />
+                ))} </>)
+            }
```

Comment:
I don't think this is quite what Charlie was after here - rather than having two pieces of logic here (No emails and then emails) you could combine the display of both NoEmails and EmailPreview into a single ternary statement.

condition ? thing to display if true : thing to display if false